doc/lua_api.md

```diff
@@ -8250,7 +8250,13 @@ child will follow movement and rotation of that bone.
     object.
 * `set_detach()`: Detaches object. No-op if object was not attached.
 * `set_bone_position([bone, position, rotation])`
-    * Shorthand for `set_bone_override(bone, {position = position, rotation = rotation:apply(math.rad)})` using absolute values.
+    * Sets absolute bone overrides, e.g. it is equivalent to
+      ```lua
+      obj:set_bone_override(bone, {
+          position = {vec = position},
+          rotation = {vec = rotation:apply(math.rad)}
```

Comment:
> Sets absolute bone overrides, e.g. it is equivalent to

but this code would set a relative bone override if I understand correctly?

Reply:
Good catch. (In hindsight I would have preferred the opposite default, but it's too late to change that now.) Fixed.